util/FINETUNE_LM_PARAMETERS.md

```diff
@@ -0,0 +1,43 @@
+# Finetune Language Model Parameters: Alpha, Beta
+This topic is to explain how did `finetune_lm_params.py` fit `alpha` and `beta`
+
+## Inspiration
+According to mesh plot result of [Paddle/DeepSpeech-Grid-Search](https://github.com/PaddlePaddle/DeepSpeech/raw/develop/docs/images/tuning_error_surface.png), I assume `BatchWER v.s. Alpha` is `ideal valley parabolic curve`, i.e. which fits this formula below:
```

Comment:
This will be true near the minimum of the BatchWER as a function of Alpha graph (the first derivative vanishes and higher derivatives are dominated by the second order derivative), but away from the minimum I see no reason for the quadratic approximation to hold in general, though it may.